src/base/serial/sermouse.c

```diff
@@ -104,6 +104,11 @@ static void ser_mouse_move_buttons(int lbutton, int mbutton, int rbutton,
   add_buf(com, buf, sizeof(buf));
 }
 
+static void ser_mouse_move_wheel(int dy, void *udata)
+{
+  dosemu_error("serial wheel mouse\n");
```

Comment:
I'd like to have PS/2 and serial modes supported as well before a merge, but maybe it's better to start with just int33.